### ISSUE TITLE
Changing wiki.style to "file-stem" seems to have no effect on the completions generated

### ISSUE THREAD

**Issue by Feel-ix-343** [artempyanykh/marksman#247]
Using neovim with version 2023-7-25

here is my .marksman.toml

```
[code_action]
# Enable/disable "Table of Contents" code action
toc.enable = false

[completion]
# The style of wiki links completion.
# Other values include: 
# * "file-stem" to complete using file name without an extension,
# * "file-path-stem" same as above but using file path.
wiki.style = "file-stem"
```

located in project root. 

When the completion is selected, it will use the file name, but in the preview, it is only using the `#{heading}` in the file. 

I would prefer that the file name is shown in the preview.

**Comment by Feel-ix-343** [comment]
..

**Comment by artempyanykh** [comment]
Hey @Feel-ix-343. So, what you're saying is that the *inserted* completions are correct, but in the preview you want *label* and *detail* to switch places depending on the completion style (see img) ?
<img width="374" alt="Screenshot 2023-09-15 at 10 13 49" src="https://github.com/artempyanykh/marksman/assets/2338443/55b2afea-72f5-460a-8ac1-5df12ea6da55">

**Comment by Feel-ix-343** [comment]
Right. 

Right now I cant actually search by file name because the search runs on the first heading name

**Comment by artempyanykh** [comment]
Yeah, ok. My workflow's a bit different but I see how this can makes sense.
Anyhow, I'm a bit busy now and it may take a while before I get to this. But if somebody submits a PR meantime I'd be happy to review it! The changes should be fairly straightforward.

**Comment by silverhook** [comment]
@artempyanykh, got any tips where in the code a newcomer can start looking at this?
